### PR TITLE
[WFLY-8862] Corrected schema version in several .xsd files

### PR DESCRIPTION
**Pull request by apintea**
Jira: https://issues.jboss.org/browse/WFLY-8862
This PR may need WildFly Core changes: https://github.com/wildfly/wildfly-core/pull/2509

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -217,7 +217,7 @@
         <version.org.wildfly.arquillian>2.1.0.Alpha1</version.org.wildfly.arquillian>
         <version.org.wildfly.build-tools>1.2.2.Final</version.org.wildfly.build-tools>
         <version.org.wildfly.checkstyle-config>1.0.5.Final</version.org.wildfly.checkstyle-config>
-        <version.org.wildfly.core>3.0.0.Beta24</version.org.wildfly.core>
+        <version.org.wildfly.core>3.0.0.Beta25</version.org.wildfly.core>
```
Comment (kabir) [inline pom.xml]: You should not be upgrading the version of wildfly core here

Comment (apintea) [inline pom.xml]: But does the version of WF Core need an update somewhere (as a similar case/situation)?

Comment (kabir) [inline pom.xml]: No. If you rebase your PR you will see that core is already on Beta26. Your core PR will be part of a later core release

**Comment by apintea** [comment]
@kabir Hi, sorry for the messy work. Here is the clean PR: https://github.com/wildfly/wildfly/pull/10227

Thank you.

**Comment by kabir** [comment]
Superseded by https://github.com/wildfly/wildfly/pull/10227

**Pull request closed without merging**